hooks/boot/05-data-helpers.rb

```diff
@@ -0,0 +1,138 @@
+require 'fileutils'
+
+module DataResetHookContextExtension
```

Comment:
I've started to think we shouldn't do have this. Rather, all data reset functionality should be in `hooks/pre/10-reset_data.rb`. We should not use these things in other hooks.

Another effect of moving the pulp reset hook handling in that file is that you can prevent the double mongo clear if `--reset-data` and `--clear-pulp-content` are both passed in with a simple `if/else`.